docs/A11y-Testing-Cross-Device.md

```diff
@@ -0,0 +1,12 @@
+# Testing on Multiple Devices for an [A11y](https://www.a11yproject.com/about/#what-does-the-term-a11y-mean) Swarm
```

Comment:
Please update to "How to test on multiple devices during development for accessibility"